Error building the project from the source

### Expected behavior
Should be able to compile the project from the source

### Actual behavior
```
╭─jhoarau@CLI-26003 ~/src/misc/reactor/reactor-core  ‹master*›
╰─➤  ./gradlew build
Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'reactor'.
> Could not resolve all files for configuration ':classpath'.
   > Could not find org.jetbrains.dokka:integration:0.9.15.
     Searched in the following locations:
         file:/Users/jhoarau/.m2/repository/org/jetbrains/dokka/integration/0.9.15/integration-0.9.15.pom
         file:/Users/jhoarau/.m2/repository/org/jetbrains/dokka/integration/0.9.15/integration-0.9.15.jar
         https://packages.atlassian.com/maven/repository/internal/org/jetbrains/dokka/integration/0.9.15/integration-0.9.15.pom
         https://packages.atlassian.com/maven/repository/internal/org/jetbrains/dokka/integration/0.9.15/integration-0.9.15.jar
         http://repo.spring.io/plugins-release/org/jetbrains/dokka/integration/0.9.15/integration-0.9.15.pom
         http://repo.spring.io/plugins-release/org/jetbrains/dokka/integration/0.9.15/integration-0.9.15.jar
         https://plugins.gradle.org/m2/org/jetbrains/dokka/integration/0.9.15/integration-0.9.15.pom
         https://plugins.gradle.org/m2/org/jetbrains/dokka/integration/0.9.15/integration-0.9.15.jar
     Required by:
         project : > org.jetbrains.dokka:dokka-gradle-plugin:0.9.15

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 43s
```

### Workaround
Updating to `org.jetbrains.dokka:0.9.16` seems to fix the issue

--- a/build.gradle
+++ b/build.gradle
@@ -33,7 +33,7 @@ buildscript {
 plugins {
   id 'org.asciidoctor.convert' version '1.5.6'
   id "me.champeau.gradle.jmh" version "0.4.4" apply false
-  id "org.jetbrains.dokka" version "0.9.15"
+  id "org.jetbrains.dokka" version "0.9.16"
   id "me.champeau.gradle.japicmp" version "0.2.6"
 }
 

